defi/src/cli/fillLast.ts

```diff
@@ -26,11 +27,14 @@ const main = async () => {
     false,
     true,
     true,
+    undefined,
+    { overwriteExistingData: true }
   );
-  console.log("TVL", typeof tvl === "number" ? humanizeNumber(tvl):tvl)
+  console.log("TVL", typeof tvl === "number" ? humanizeNumber(tvl) : tvl)
 };
 
-main().then(() => {
+main().then(async () => {
   console.log('Done!!!')
+  await closeConnection()
```

Comment:
i think connection will be closed anyway with process.exit() but its ok, doesnt matter anyway

Reply:
issue here was, importAdapter was taking too long, in ts, it was like 10 minutes on my laptop, with this js, it was 9 seconds
I dunno how to get around ts taking so long for each adapter import

Reply:
the issue is that importAdapter tries to import a file with all adapters, which ofc is really slow cause it needs to parse and execute a ton of js code, the reason we do this is because esbuild doesnt support dynamic imports, but its completely unnecessary when you just need to import a single adapter

If you look at the importAdapter() code I actually left a comment  on how to speed it up when using it locally, you just need to delete the line `import adapters from "./adapters"` and replace the return statement in the function with a dynamic import : ```
return import(`@defillama/adapters/projects/${protocol.module}`)
```

then the full file looks like this
```
import type { Protocol } from "../../protocols/types";

export function importAdapter(protocol: Protocol) {
    return import(`@defillama/adapters/projects/${protocol.module}`)
}
```

Reply:
but in this case, I do want to import all the adapters, just that it is slow in the ts

Reply:
yeah in that case rip, maybe the solution would be to first compile with esbuild and then execute the compiled code, instead of using ts-node, that should be much faster (albeit still not super fast)